packages/flutter/lib/src/material/page.dart

```diff
@@ -267,12 +98,12 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   Widget buildTransitions(BuildContext context, Animation<double> animation, Animation<double> forwardAnimation, Widget child) {
     if (Theme.of(context).platform == TargetPlatform.iOS &&
         Navigator.of(context).userGestureInProgress) {
-      return new _CupertinoPageTransition(
+      return new CupertinoPageTransition(
         animation: new AnimationMean(left: animation, right: forwardAnimation),
         child: child
```

Comment:
I'm a bit worried that the background color on the transition will cause a flash if the page doesn't paint an opaque background, but maybe it is required to do that anyway?